Web/Edubase.Services/Enums/EnumSets.cs

```diff
@@ -36,7 +36,8 @@ public static class EnumSets
             ET.FreeSchoolsAlternativeProvision,
             ET.FreeSchoolsSpecial,
             ET.StudioSchools,
-            ET.UniversityTechnicalCollege
+            ET.UniversityTechnicalCollege,
+            ET.AcademySecure16to19
```

Comment:
This appears to not have any references in other parts of the code.  So it maybe is not used.  Talked with Mark and added this because it is a list containing academies.